pkg/ui/workspaces/cluster-ui/tsconfig.json

```diff
@@ -17,7 +17,6 @@
     "baseUrl": ".",
     "paths": {
       "@cockroachlabs/crdb-protobuf-client": [
-        "../../../../_bazel/bin/pkg/ui/workspaces/db-console/src/js",
```

Comment:
Let's pull this back in as `../../../../_bazel/bin/pkg/ui/workspaces/db-console/src/js/crdb-protobuf-client`. This file is used for editors as well as compilation, to configure the TypeScript language server. Since `rules_js` doesn't hoist compiled JS out of the bazel output tree and back into the workspace (where the `node_modules/` symlink would point to the correct location), we have to tell TypeScript where to look for it. Having two elements here sets a queue of locations to check, and the first found location wins. In short, this allows `./dev gen js` to get a working editor environment set up :)